.github/workflows/release-aur.yml

```diff
@@ -0,0 +1,208 @@
+
+      - name: Upload Binary PKGBUILD Artifact
+        uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4.6.2
+        with:
+          name: pkgbuild-bin
+          path: aur-package-bin/PKGBUILD
+          retention-days: 30
+
+      - name: Publish Binary AUR Package
+        uses: KSXGitHub/github-actions-deploy-aur@2ac5a4c1d7035885d46b10e3193393be8460b6f1 #4.1.1
+        with:
+          pkgname: usb-tree-app-bin
+          pkgbuild: ./aur-package-bin/PKGBUILD
+          commit_username: ${{ secrets.AUR_USERNAME }}
+          commit_email: ${{ secrets.AUR_EMAIL }}
+          ssh_private_key: ${{ secrets.AUR_SSH_PRIVATE }}
+          commit_message: "chore: update AUR package to ${{ env.version }}"
+          ssh_keyscan_types: rsa,ecdsa,ed25519
+
+  update-aur-source:
+    name: Update AUR Source Package
+    runs-on: ubuntu-latest
+
+    container:
+      image: archlinux
+
+    steps:
+      - name: Install Dependencies
+        run: |
+          pacman -Syu --noconfirm git openssh jq base-devel
+
+      - name: Get Release Information
+        run: |
+          latest_version=$(curl -s https://api.github.com/repos/AOzmond/usb-tree/releases/latest | jq -r .tag_name)
+
+          # Download the source tarball from the release tag
+          curl -L https://github.com/AOzmond/usb-tree/archive/refs/tags/${latest_version}.tar.gz -o usb-tree-source.tar.gz
+          sha256sum_source=$(sha256sum usb-tree-source.tar.gz | cut -d' ' -f1)
+
+          echo "VERSION: $latest_version"
+          echo "SHA256: $sha256sum_source"
+          echo "version=$latest_version" >> $GITHUB_ENV
+          echo "sha256sum_source=$sha256sum_source" >> $GITHUB_ENV
+
+      - name: Create PKGBUILD Directory
+        run: |
+          mkdir -p aur-package-src
+
+      - name: Generate PKGBUILD for Source Package
+        run: |
+          cat > aur-package-src/PKGBUILD << 'PKGBUILD_EOF'
+          # Maintainer: Alastair Ozmond <alastair.ozmond@gmail.com>
+          pkgname=usb-tree-app
+          pkgver=VERSION_PLACEHOLDER
+          pkgrel=1
+          pkgdesc="USB device tree viewer with detailed information"
+          arch=('x86_64')
+          url="https://github.com/AOzmond/usb-tree"
+          license=('GPL-2.0-or-later')
+          depends=('gtk3' 'webkit2gtk-4.1' 'libusb')
+          makedepends=('go' 'bun-bin' 'git')
+          provides=('usb-tree')
+          conflicts=('usb-tree' 'usb-tree-app-bin')
+          source=("${pkgname}-${pkgver}.tar.gz::${url}/archive/refs/tags/${pkgver}.tar.gz")
+          sha256sums=('SHA256_PLACEHOLDER')
+
+          build() {
+              cd "${srcdir}/usb-tree-${pkgver}"
+
+              # Install frontend dependencies
+              cd app/frontend
+              bun install
+
+              # Build the application with wails
+              cd ../
+              go install github.com/wailsapp/wails/v2/cmd/wails@latest
+              ~/go/bin/wails build -clean -platform linux/amd64
```

Comment:
`go install` does not guarantee installation to `~/go/bin`, it is `$(go env GOPATH)/bin` (often `$GOPATH` but not always set)

Reply:
Do you need to limit this package to x86_64?